lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -166,6 +166,12 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
       final path =
           _generatePolygonPath(centerX, centerY, radius, data.titleCount);
 
+      /// draw radar shadow
+      if (data.elevation > 0) {
+        canvasWrapper.canvas
```

Comment:
I see that you're adding a shadow / elevation here.
But the PR title is `add gradient to RadarChart`

So please open separate PRs for different featuers/improvements.